docs/sources/configure/examples.md

```diff
@@ -225,70 +189,83 @@ storage_config:
 
 schema_config:
   configs:
-    # Starting from 2018-04-15 Loki should store indexes on Cassandra
+    # Starting from 2018-04-15 Loki should store indexes on BoltDB
     # using weekly periodic tables and chunks on filesystem.
     # The index tables will be prefixed with "index_".
   - from: "2018-04-15"
-    store: cassandra
+    store: boltdb-shipper
     object_store: filesystem
-    schema: v11
+    schema: v12
     index:
         period: 168h
         prefix: index_
 
-  # Starting from 2020-6-15 we moved from filesystem to AWS S3 for storing the chunks.
-  - from: "2020-06-15"
-    store: cassandra
+  # Starting from 2023-6-15 Loki should store indexes on TSDB
+  # using daily periodic tables and chunks on AWS S3.
+  # The schema version is also bumped to v13.
```

Comment:
"The schema version is also bumped to v13" Is this a typo? Line 209 still says `schema: v12` and all the other examples say `v12`.

Reply:
Fixed.